tests/test_samplers.py

```diff
@@ -109,12 +112,12 @@ def test_external_embedding_sampler(self):
         # verify mock sampler received custom kwargs
         self.assertEqual(res.subsamples.first.energy, -1)
 
-    def test_clique_embedder(self):
+    @parameterized.expand([['chimera', 2], ['pegasus', 1]])
```

Comment:
Can you add a test for Zephyr as well?

Reply:
Now I see your comment above. You can up the lower bound in `tests/requirements.txt` only.